add rule for application access after rabbitmq network policy update (#1043)
elastisys/welkin: docs/user-guide/additional-services/rabbitmq.md

- [Create a Pod that has access to the secret data through a Volume](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#create-a-pod-that-has-access-to-the-secret-data-through-a-volume)
- [Define container environment variables using Secret data](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#define-container-environment-variables-using-secret-data)

## Allow your Pods to communicate with the RabbitMQ cluster

> The RabbitMQ cluster is protected by Network Policies. To allow your application communicating with the cluster, add the following label to your Pods: `elastisys.io/rabbitmq-<cluster_name>-access: allow`

Reviewer: Nice to see this being added
Small nit apart from the one Andre has pointed with 1 sentence per line.
I would rephrase this `To allow your application communicating with the cluster, add the following label to your Pods:` to something like this `To allow your application to communicate with the RabbitMQ cluster, add the following label to your application Pods:`

> ## Allow your Pods to communicate with the RabbitMQ cluster
> 
> The RabbitMQ cluster is protected by Network Policies. To allow your application communicating with the cluster, add the following label to your Pods: `elastisys.io/rabbitmq-<cluster_name>-access: allow`
> 
> You can get `cluster_name` from your administrator.
> 

Reviewer 2: ```suggestion
### Allow your Pods to communicate with the RabbitMQ cluster

The RabbitMQ cluster is protected by Network Policies.
To allow your application communicating with the cluster, add the following label to your Pods: `elastisys.io/rabbitmq-${RABBITMQ_CLUSTER}-access: allow`

Using the `${RABBITMQ_CLUSTER}` from before, you can also get it from your administrator.
```

I think we should harmonise it into the accessing rabbitmq cluster section.

- [Create a Pod that has access to the secret data through a Volume](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#create-a-pod-that-has-access-to-the-secret-data-through-a-volume)
- [Define container environment variables using Secret data](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#define-container-environment-variables-using-secret-data)

### Allow your Pods to communicate with the RabbitMQ cluster

> 

Reviewer 3: ```suggestion
!!!important "Important: Access Control with NetworkPolicies"
```

> The RabbitMQ cluster is protected by Network Policies.

Reviewer 3: ```suggestion
The RabbitMQ cluster is protected by [Network Policies](../safeguards/enforce-networkpolicies.md).
```
